data/blog/linux-32bit-support.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Discontinuing support for 32-bit Linux
+author: felixrieseberg
+date: '2019-03-15'
+---
+
+The Electron team will discontinue support for 32-bit/i386 Linux starting with Electron v4.0. The last version of Electron that supports 32-bit based installations of Linux is Electron v3.1, which will receive support releases until Electron v6 is released. We will continue to support 64-bit based Linux (x64).
+
+---
+
+## What exactly is Electron no longer supporting?
+
+You may have seen the description "64-bit" and "32-bit" as stickers on your computer or as options for downloading software. The term is used to describe a specific computer architecture. Most computers made in the 1990s and early 2000s were made with CPUs that were based on the 32-bit architecture, while most computers made later were based on the newer and more powerful 64-bit architecture. The Nintendo 64 (get it?) and the PlayStation 2 were the first widely available consumer devices with the new architecture, computers sold after 2010 contained almost exclusively 64-bit processors. As a result, support has been shrinking: Google has stopped releasing Chrome for 32-bit Linux back in March 2016, Canonical has stopped providing 32-bit desktop images in 2017, dropping support for 32-bit altogether with Ubuntu 18.10. Arch Linux, elementary OS, and other prominent Linux distributions have already dropped support for the aging processor architecture.
+
+Until now, Electron has provided and supported builds that run on the older 32-bit architecture. From release v4.0 onwards, the Electron team will no longer be able to provide binaries or support for 32-bit Linux.
+
+Electron has always been a vibrant open source project and we continue to support and encourage developers interested in building Electron for exotic architectures.
+
+## What does that mean for developers?
+
+If you are not currently providing 32-bit distributions of your app for Linux, no action is required.
+
+If you are building a cross-platform application using Electron and are currently providing 32-bit distributions of your app for Linux, we recommend that you too drop support for 32-bit Linux installations. No changes are required to your application's code.
```

Comment:
```suggestion
Projects which ship 32-bit Linux Electron applications will need to decide how to proceed. 32-bit Linux will be supported on Electron 3 [until](https://electronjs.org/docs/tutorial/support#supported-versions) the release of Electron 6, which gives some time to make decisions and plans.
```

This suggestion removes "we recommend that you too drop support" because that phrasing was harsh and because it feels slightly inappropriate to tell downstream what to do.